ee/server/service/software_installers.go

```diff
@@ -1006,12 +1006,21 @@ func (svc *Service) InstallSoftwareTitle(ctx context.Context, hostID uint, softw
 }
 
 func (svc *Service) installSoftwareFromVPP(ctx context.Context, host *fleet.Host, vppApp *fleet.VPPApp, appleDevice bool, selfService bool) (string, error) {
+	token, err := svc.GetVPPTokenIfCanInstallVPPApps(ctx, appleDevice, host)
```

Comment:
Curious: why the split out into these 2 methods?

Reply:
Oh I see the InstallVPPAppPostValidation is used in new functionality with that new policyID param,
VSCode just wasn't smart enough to include that in the references.